content/tutorials/use-simple-xrp-payments/send-xrp.ja.md

```diff
@@ -114,54 +81,30 @@ async function doPrepare() {
     "maxLedgerVersionOffset": 75
   })
   const maxLedgerVersion = preparedTx.instructions.maxLedgerVersion
-  console.log("Prepared transaction instructions:", preparedTx.txJSON)
-  console.log("Transaction cost:", preparedTx.instructions.fee, "XRP")
-  console.log("Transaction expires after ledger:", maxLedgerVersion)
+  console.log("準備されたトランザクション指示：", preparedTx.txJSON)
+  console.log("トランザクションコスト：", preparedTx.instructions.fee, "XRP")
+  console.log("トランザクションの有効期限はこのレジャー後：", maxLedgerVersion)
```

Comment:
Rather than use the include_code filter here, I opted to leave the code samples inlined in the Markdown file but translated. Since the text is different, there's going to be a redundant (possibly out of sync) copy regardless, but I hope it's more helpful if the log messages at least are translated.